ledger/cryptarchia-ledger/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "cryptarchia-ledger"
```

Comment:
curious, why a separate crate and not a  `ledger` module?

Reply:
it was already a ledger module before 🤔 
The idea is to make it clear we would like to have a distinction between the two, in the sense that consensus should have limited influence on ledger choices (e.g. transactions should probably have the same format regardless of the consensus alg, they should work with both carnot and cryptarchia). As a result, those items might better be defined in their own separate crate (for now I've not created a common crate/module since we don't have those components yet).

Of course, some things will have to change (like the header format and leadership proof), and for this reason we have specific ledger modules/crates.